Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/connected_components.h

```diff
@@ -805,9 +808,10 @@ void remove_connected_components(PolygonMesh& pmesh
 *  keeps the connected components not designated by the faces in `components_to_remove`,
 *  and removes the other connected components and all isolated vertices.
 *
-* Property maps for `CGAL::face_index_t` and `CGAL::vertex_index_t`
-* must be either available as internal property maps
-* to `pmesh` or provided as \ref pmp_namedparameters "Named Parameters".
+* If `PolygonMesh`
+*  has a non writable internal property map
+*  for `CGAL::face_index_t` or `CGAL::vertex_index_t` and no `face_index_map` (respectively `vertex_index_map`) is given
+*  as a named parameter, then the internal one(s) must be initialized. Otherwise, it will be.
```

Comment:
```suggestion
*  for `CGAL::face_index_t` (resp. `CGAL::vertex_index_t`) and no `face_index_map` (resp. `vertex_index_map`) is given
*  as a named parameter, then the internal one must be initialized. Otherwise, it will be.
```